src/components/forms/FileInput/FileInput.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react'
+import classnames from 'classnames'
+
+import { FilePreview } from './FilePreview'
+import { makeSafeForID } from './utils'
+
+interface FileInputProps {
+  id: string
+  name: string
+  disabled?: boolean
+  multiple?: boolean
+  accept?: string
```

Comment:
It would be nice for the type of `accept?` to be `string | string[]` to make it more obvious how to pass multiple options and avoid needing to split on a string inside the component. I can imagine these use cases:

```jsx
<FileInput accepts="pdf" ... />
<FileInput accepts={["pdf", "docx", "png"]} ... />
```

The second one needs a little more syntax than the current `accept="pdf,docx,png"`, but I think having the type system help a developer provide the options correctly outweighs that.

Reply:
hmm yeah, I think the default string syntax is due to this attribute being passed to the `<input />` (and that's the [syntax for the native `accept` attr](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#attr-accept)) but I could see allowing an array of strings being a nice enhancement. In that case we'd also need to join them into a single string to pass into the `<input />` so there's some extra handling either way.

Reply:
I didn't realize that `input` had an `accept` attribute natively. In that case it's probably fine as-is, at least in terms of being consistent with how we handle other things in the library. Thanks for the thoughtful response!